nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1451,13 +1448,13 @@ def _run_connected_components(
             self.profile_dir, "connected-components-run"
         ):
 
-            Comms.initialize(p2p=True)
+            Comms.initialize(p2p=False)
             df = dask_cudf.read_parquet(
                 deduped_encoded_jaccard_path, blocksize="1GB", aggregate_files=True
             )
             df = df[df["jaccard"] == 1].reset_index(drop=True)
 
-            labels_df = dask_cudf.read_parquet(deduped_parsed_id_path)
+            labels_df = dask_cudf.read_parquet(deduped_parsed_id_path, blocksize="1GB", aggregate_files=True)
```

Comment:
This was causing errors as we had too little files and it some caused unequal distribution of work.  Testing again to verify this is robust